core/src/main/java/org/apache/accumulo/core/rpc/SslConnectionParams.java

```diff
@@ -141,17 +141,17 @@ public static SslConnectionParams forClient(AccumuloConfiguration configuration)
   private static String findKeystore(String keystorePath) throws FileNotFoundException {
     try {
       // first just try the file
-      File file = Path.of(keystorePath).toFile();
-      if (file.exists()) {
-        return file.getAbsolutePath();
+      Path path = Path.of(keystorePath);
+      if (Files.exists(path)) {
+        return path.toAbsolutePath().toString();
```

Comment:
This is a private method. It can return the absolute path without converting it to a string. The calling code can keep it as a path or convert it as needed. That reduces the assumptions this code makes on how it is used by the callers.